services/app-api/src/emailer/emails/sendQuestionStateEmail.test.ts

```diff
@@ -0,0 +1,320 @@
+    submissionType: 'CONTRACT_AND_RATES',
+    riskBasedContract: false,
+    submissionDescription: 'A submitted submission',
+    supportingDocuments: [
+        {
+            s3URL: 'bar',
+            name: 'foo',
+            sha256: 'fakesha',
+        },
+    ],
+    contractType: 'BASE',
+    contractExecutionStatus: undefined,
+    contractDocuments: [
+        {
+            s3URL: 'bar',
+            name: 'foo',
+            sha256: 'fakesha',
+        },
+    ],
+    contractDateStart: new Date('01/01/2024'),
+    contractDateEnd: new Date('01/01/2025'),
+    managedCareEntities: ['MCO'],
+    federalAuthorities: ['VOLUNTARY', 'BENCHMARK'],
+    inLieuServicesAndSettings: undefined,
+    modifiedBenefitsProvided: undefined,
+    modifiedGeoAreaServed: undefined,
+    modifiedMedicaidBeneficiaries: undefined,
+    modifiedRiskSharingStrategy: undefined,
+    modifiedIncentiveArrangements: undefined,
+    modifiedWitholdAgreements: undefined,
+    modifiedStateDirectedPayments: undefined,
+    modifiedPassThroughPayments: undefined,
+    modifiedPaymentsForMentalDiseaseInstitutions: undefined,
+    modifiedMedicalLossRatioStandards: undefined,
+    modifiedOtherFinancialPaymentIncentive: undefined,
+    modifiedEnrollmentProcess: undefined,
+    modifiedGrevienceAndAppeal: undefined,
+    modifiedNetworkAdequacyStandards: undefined,
+    modifiedLengthOfContract: undefined,
+    modifiedNonRiskPaymentArrangements: undefined,
+    statutoryRegulatoryAttestation: undefined,
+    statutoryRegulatoryAttestationDescription: undefined,
+    stateContacts: [
+        {
+            name: 'test1',
+            titleRole: 'Foo1',
+            email: 'test1@example.com',
+        },
+        {
+            name: 'test2',
+            titleRole: 'Foo2',
+            email: 'test2@example.com',
+        },
+    ],
+}
+const dateAsked = new Date('01/01/2024')
+test('to addresses list includes submitter emails', async () => {
+    const sub = mockContractRev()
+    const defaultStatePrograms = mockMNState().programs
+    const template = await sendQuestionStateEmail(
+        sub,
+        defaultSubmitters,
+        cmsUser,
+        testEmailConfig(),
+        defaultStatePrograms,
+        dateAsked
+    )
+
+    if (template instanceof Error) {
+        throw template
+    }
+
+    expect(template).toEqual(
+        expect.objectContaining({
+            toAddresses: expect.arrayContaining(defaultSubmitters),
+        })
+    )
+})
+
+test('to addresses list includes all state contacts on submission', async () => {
+    const sub: ContractRevisionWithRatesType = {
+        ...mockContractRev({ formData }),
+    }
+    const defaultStatePrograms = mockMNState().programs
+    const template = await sendQuestionStateEmail(
+        sub,
+        defaultSubmitters,
+        cmsUser,
+        testEmailConfig(),
+        defaultStatePrograms,
+        dateAsked
+    )
+
+    if (template instanceof Error) {
+        console.error(template)
```

Comment:
there's a bunch more of these instances where you're returning an error instead of throwing them, can you fix those too?